katcp/resource.py

```diff
@@ -894,9 +892,7 @@ def handle_update(sensor, reading):
             # This handler is called whenever a sensor update is received
             try:
                 assert sensor is self
-                cond_matched = condition_test(reading)
-                status_matched = reading.status == status or status is None
```

Comment:
Agree, I wanted to make sure we did not want to add status checking in the non-callable case as well. Given that we agree that the satus-checking is not necessary its superflous to discuss this anymore.